src/Akeneo/Platform/Installer/back/src/Infrastructure/Symfony/Resources/config/routing.yml

```
@@ -11,3 +11,10 @@ akeneo_installer_check_reset_instance:
     _controller: Akeneo\Platform\Installer\Infrastructure\Controller\CheckInstanceCanBeResetAction
     _feature: reset_pim
   methods: [ GET ]
+
+akeneo_installer_is_maintenance_mode_enabled:
+  path: /rest/is-maintenance-mode_enabled
```

```suggestion
  path: /rest/is-maintenance-mode-enabled
```
But I agree with @StevenVAIDIE, we may not need this after all

---

:+1:  I'll keep it for now and delete it in front PR if I'm not using it